src/components/basic-element/basic-element.less

```diff
@@ -73,6 +73,15 @@
       //This line can probably be removed eventually, as box-shadow is being applied in the component properties now.
       //But removing it might affect existing projects, so hesitant to do so quite yet.
       box-shadow: inset 0px 2px 0px rgba(255,255,255, 0.3);
+      display: flex;
+      align-items: center;
+
+      img {
+        top: 1px;
+        opacity: 0.8;
+        height: 100%;
+        margin: 0 0 0 5px;
```

Comment:
I suggest adding `margin: 0 0 0 8px` to give it a bit more spacing. It feels just a bit tight at the moment.